tests/evpn_tunnel.py

```diff
@@ -0,0 +1,1143 @@
+
+        # create the VXLAN tunnel Term entry in Config DB
+        self.helper.create_entry_tbl(
+            conf_db,
+            "VXLAN_TUNNEL", name,
+            attrs,
+        )
+
+    def create_vxlan_tunnel_map(self, dvs, tnl_name, map_name, vni_id, vlan_id):
+        conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+        attrs = [
+                ("vni", vni_id),
+                ("vlan", vlan_id),
+        ]
+
+        # create the VXLAN tunnel Term entry in Config DB
+        self.helper.create_entry_tbl(
+            conf_db,
+            "VXLAN_TUNNEL_MAP", "%s|%s" % (tnl_name, map_name),
+            attrs,
+        )
+
+    def create_evpn_remote_vni(self, dvs, vlan_id, remote_vtep, vnid):
+        app_db = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+        self.helper.create_entry_pst(
+            app_db,
+            "VXLAN_REMOTE_VNI_TABLE", "%s:%s" % (vlan_id, remote_vtep),
+            [
+                ("vni", vnid),
+            ],
+        )
+        time.sleep(2)
+
+    def remove_vxlan_tunnel(self, dvs, tnl_name):
+        conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+        # create the VXLAN tunnel Term entry in Config DB
+        self.helper.delete_entry_tbl(
+            conf_db,
+            "VXLAN_TUNNEL", tnl_name,
+        )
+
+    def remove_vxlan_tunnel_map(self, dvs, tnl_name, map_name,vni_id, vlan_id):
+        conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+        attrs = [
+                ("vni", vni_id),
+                ("vlan", vlan_id),
+        ]
+
+        # create the VXLAN tunnel Term entry in Config DB
+        self.helper.delete_entry_tbl(
+            conf_db,
+            "VXLAN_TUNNEL_MAP", "%s|%s" % (tnl_name, map_name),
+        )
+
+    def remove_evpn_remote_vni(self, dvs, vlan_id, remote_vtep ):
+        app_db = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+        self.helper.delete_entry_pst(
+            app_db,
+            "VXLAN_REMOTE_VNI_TABLE", "%s:%s" % (vlan_id, remote_vtep),
+        )
+        time.sleep(2)
+
+    def create_vxlan_vrf_tunnel_map(self, dvs, vrfname, vni_id):
+        conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+        attrs = [
+                ("vni", vni_id),
+        ]
+
+        # create the VXLAN VRF tunnel Term entry in Config DB
+        self.helper.create_entry_tbl(
+            conf_db,
+            "VRF", vrfname,
+            attrs,
+        )
+
+    def remove_vxlan_vrf_tunnel_map(self, dvs, vrfname):
+        conf_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+
+        attrs = [
+                ("vni", "0"),
+        ]
+
+        # remove the VXLAN VRF tunnel Term entry in Config DB
+        self.helper.create_entry_tbl(
+            conf_db,
+            "VRF", vrfname,
+            attrs,
+        )
+
+    def create_vlan1(self, dvs, vlan_name):
```

Comment:
Looks like a typo: probably should be ```create_vlan``` instead of ```create_vlan1```

Reply:
There is another function create_vlan which has a different purpose. Again this is a refactored code and so I just moved it from one file to another.